### DISCUSSION TITLE
Change the UI for switching workspaces to a HTML / CSS one like the rest of the floorp UI

### DISCUSSION THREAD

**Discussion by YousufSSyed** [Floorp-Projects/Floorp#435] [Feature Request]
Right now the menu looks very unpleasant, with or without my CSS theme, and its harder to apply CSS to it than if it used the same UI elements as the rest of Firefox / Floorp. 
<img width="247" alt="Screenshot 2023-08-30 at 8 48 05 PM" src="https://github.com/Floorp-Projects/Floorp/assets/42646062/fce6a048-01cd-4ec4-b8a1-642c8f4138f4">

**Comment by menndouyukkuri** [comment]
Isn't changing the structure and changing the look of the structure two different things?

**Reply by YousufSSyed** [reply]
@menndouyukkuri I didn't say anything about structure, maybe it's a translation issue on your end? Anyway, I think the menu should look cleaner, like have the workspaces consistently sized, and to have just one button per row instead of two.
